lib/wraft_doc/document/document.ex

```diff
@@ -66,7 +66,7 @@ defmodule WraftDoc.Document do
       {:ok, layout} ->
         layout = layout_files_upload(layout, params)
         fetch_and_associcate_assets(layout, current_user, params)
-        Repo.preload(layout, [:engine, :creator, :assets])
+        Repo.preload(layout, [:engine, :creator, :assets, :frame])
```

Comment:
If you are adding here frame, it won't be listed in the layout api unless you specify it in layout view. So in that case the user. Don't you think we need to add it as part of layout view?